external-api/src/interfaces/premiumSchedule.interface.ts

```diff
@@ -0,0 +1,14 @@
+export interface PremiumSchedule {
```

Comment:
From what I read in the other file all of these are always provided and not nullable, but if it's safer I'm happy to make these all optional parameters in the interface